variables.tf

```diff
@@ -21,68 +21,60 @@ variable "users" {
   default = {}
 }
 
-variable "SES_notifier" {
-  sensitive = true
-  type = object({
-    enabled = bool
-    parameters = object({
-      SOURCE_EMAIL_ADDRESS = string
-      DEST_EMAIL_ADDRESS   = string
-      zone_id              = string
+variable "SES_notifiers" {
+  type = list(object({
+    zone_id = string
+    py_params = object({
+      source_email_address = string
+      dest_email_address   = string
     })
-  })
-  default = {
-    enabled    = false
-    parameters = null
-  }
+  }))
+  default = []
 }
 
-variable "Slack_notifier" {
-  sensitive = true
-  type = object({
-    enabled = bool
-    parameters = object({
-      WEBHOOK = string
+variable "Slack_notifiers" {
+  type = list(object({
+    py_params = object({
+      webhook = string
     })
```

Comment:
For all notifiers,
We don't add a name field or an identifier so that users can differenciate several instances of the same type of notifier ?

Reply:
We left that out for now since it would not be used anywhere in the code. If only to distinguish between configurations in the config file, comments could achieve this purpose; however if we plan to add a name later, it could easier to add it now (less breaking change in the future).
Do you think we should add a `name` field now ?

Reply:
Ok, we can add it later. If it is optional, it will not be a breaking change.